includes/functions.php

```diff
@@ -3219,42 +3219,109 @@ function dokan_remove_action_button( $actions ) {
 /**
  * Dokan get translated days
  *
- * @param  string day
+ * @param  string days
  *
  * @since  2.8.2
  *
- * @return string
+ * @return string|array
  */
-function dokan_get_translated_days( $day ) {
-    switch ( $day ) {
+function dokan_get_translated_days( $days = true ) {
```

Comment:
As you have stored the `key => value` pair in the variable, it's no more necessary to check all the key manually. You can rewrite the function the following way:

```
function dokan_get_translated_days( $day = '' ) {
    $all_days = array(
        'sunday'    => __( 'Sunday', 'dokan-lite' ),
        'monday'    => __( 'Monday', 'dokan-lite' ),
        'tuesday'   => __( 'Tuesday', 'dokan-lite' ),
        'wednesday' => __( 'Wednesday', 'dokan-lite' ),
        'thursday'  => __( 'Thursday', 'dokan-lite' ),
        'friday'    => __( 'Friday', 'dokan-lite' ),
        'saturday'  => __( 'Saturday', 'dokan-lite' ),
    );

    if ( empty( $day ) ) {
        return $all_days;
    }

    if ( isset( $all_days[ $day ] ) ) {
        return $all_days[ $day ];
    }

    if ( 'close' === $day ) {
        return apply_filters( 'dokan_store_close_day_label', __( 'Off Day', 'dokan-lite' ) );
    }

    return apply_filters( 'dokan_get_translated_days', '', $day );
}
```